src/goTest.ts

```diff
@@ -122,7 +122,8 @@ function goTest(config: TestConfig): Thenable<boolean> {
 		outputChannel.show(2);
 		let buildFlags: string[] = vscode.workspace.getConfiguration('go')['buildFlags'];
 		let buildTags: string = vscode.workspace.getConfiguration('go')['buildTags'];
-		let args = ['test', '-v', '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
+		let testFlags: string[] = vscode.workspace.getConfiguration('go')['testFlags'];
+		let args = ['test', ...testFlags, '-timeout', config.timeout, '-tags', buildTags, ...buildFlags];
```

Comment:
@ramya-rao-a
>testTimeout should not be a separate setting and should be included in the testFlags setting

Hm. Wouldn't it then follow that `buildTags` should be merged into `buildFlags`? After all,  `-tag` is just another `buildFlag`. I do see value in adding convenient strongly typed wrappers around the most common flags, but those would need to be easily omitted so that users can have absolute control over flags.